benchmark_test.go

```diff
@@ -0,0 +1,23 @@
+package runn
+
+import (
+	"context"
+	"testing"
+
+	"github.com/k1LoW/runn/testutil"
+)
+
+func BenchmarkRun(b *testing.B) {
```

Comment:
@k2tzumi 
This is a sample benchmark.
Let's change it to a more valuable benchmark.